lib/suse/connect/client.rb

```diff
@@ -131,6 +132,16 @@ def system_migrations(products)
         end
       end
 
+      def run_post_register_scripts(product)
+        if File.directory?(config.post_register_scripts_path)
+          Dir["#{config.post_register_scripts_path}/*.callback"].each do |file|
+            Kernel.system file, product.identifier
+          end
+        else
+          log.debug 'Specified post-install scripts path is not a directory; skipping.'
```

Comment:
Wouldn't it help if you output what was the 'specified path' here?
It will at least help our supporters in that not-that-rare case when our customer makes a typo.